src/components/Display/CharTable.js

```diff
@@ -0,0 +1,91 @@
+import {
+  Table,
+  TableContainer,
+  TableHead,
+  TableBody,
+  TableRow,
+  TableCell,
+  Paper,
+  Avatar,
+  Box,
+} from '@mui/material';
+import React, { useState } from 'react';
+import CharModal from '../Modal/CharModal';
+
+const CharTable = ({ data }) => {
+  const [open, setOpen] = useState(false);
+  const [modalInfo, setModalInfo] = useState(null);
+
+  const handleOpen = row => {
+    setOpen(true);
+    setModalInfo(row);
+  };
+
+  return (
+    <>
+      <Box
+        sx={{
+          flexGrow: 1,
+          display: 'flex',
+        }}
+      >
+        <TableContainer component={Paper} sx={{ m: 2 }}>
+          <Table aria-label="simple table">
+            <TableHead>
+              <TableRow sx={{ backgroundColor: '#42a5f5' }}>
+                <TableCell></TableCell>
+                <TableCell sx={{ fontWeight: 'bold', color: 'white' }}>
```

Comment:
Create a `const cellSx = { fontWeight: 'bold', color: 'white' }` and use it to prevent duplication

Reply:
or use class

Reply:
done